inspirehep/ext/jinja_filters/record.py

```diff
@@ -390,3 +390,99 @@ def citation_phrase(count):
             return 'Cited 1 time'
         else:
             return 'Cited ' + str(count) + ' times'
+
+    @app.template_filter()
+    def conference_info(record):
+        out = ''
+        title_exists = False
+        if 'publication_info' in record:
+            from invenio_ext.es import es
+            for element in record['publication_info']:
+                if 'journal_title' in element:
+                    title_exists = True
+                if 'conference_recid' in element \
+                        and 'parent_recid' in element:
+                    conference_rec = es.get(
+                        index='conferences', id=element['conference_recid'])
+                    if 'journal_title' not in element:
+                        if 'page_artid' in element:
+                            if title_exists:
+                                out += '(<a href="/record/' + str(
+                                    element['parent_recid']) + \
+                                    '">Proceedings' + '</a>' + \
+                                    ' of <a href="/record/' + \
+                                    str(element['conference_recid']) + \
+                                    '">' + conference_rec['_source']['title'] + '</a>' + \
+                                    ', pages ' + element['page_artid'] + ')'
+                                break
+                            else:
+                                out += 'Published in <a href="/record/' + str(
+                                    element['parent_recid']) + \
+                                    '">proceedings' + '</a>' + \
+                                    ' of <a href="/record/' + \
+                                    str(element['conference_recid']) + \
+                                    '">' + conference_rec['_source']['title'] + '</a>' + \
+                                    ', pages ' + element['page_artid']
+                                break
+                        else:
+                            if title_exists:
```

Comment:
You have in a [previous if](https://github.com/inspirehep/inspire-next/pull/776/files#diff-50fbc1d5feafee144203f26406a1267cR407) checked that title does not exist so this case is not possible.